pkgs/development/python-modules/pysrt/default.nix

```diff
@@ -18,16 +19,22 @@ buildPythonPackage rec {
     sha256 = "1f5hxyzlh5mdvvi52qapys9qcinffr6ghgivb6k4jxa92cbs3mfg";
   };
 
-  nativeCheckInputs = [ nose ];
-  checkPhase = ''
-    nosetests -v
-  '';
+  build-system = [ setuptools ];
 
   propagatedBuildInputs = [ chardet ];
 
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  preCheck = ''
+    for file in test_srtitem test_srtfile; do
+      substituteInPlace tests/$file.py \
+        --replace-fail "assertEquals" "assertEqual"
+    done
+  '';
```

Comment:
This was actually fixed upstream; it might be better to just pin to the latest Git commit, or vendor the relevant commit. This is fine too though so I’ll merge if you don’t feel like doing that.